server/routes/api/delete/deleteUserRoutes.ts

```diff
@@ -0,0 +1,75 @@
+import { Router, Request, Response } from "express";
+import { validationResult, param } from "express-validator";
+
+import Handler from "./deleteUserHandler";
+
+const deleteRouter = Router();
+
+const handler = new Handler();
+const BABYSITTER_INVALID_INPUT_ERROR = "babysitterId must be provided and a number";
+const PARENT_INVALID_INPUT_ERROR = "parentId must be provided and a number";
+const ACTIVE_END_TIMESTAMP = '9999999999';
+
+deleteRouter.put("/parent/:parent", 
+    [ 
+        param('parent').notEmpty().isNumeric().withMessage(PARENT_INVALID_INPUT_ERROR)
+    ],
+    async (req: Request, res: Response) => {
+        try {
+            const fieldValidationResult = validationResult(req);
+            if (!fieldValidationResult.isEmpty()) {
+              return res
+                .status(400)
+                .json({ message: fieldValidationResult.array().map((item) => item.msg).join(' ') });
+            }
+            const { parent: parentId } = req.params
+            const data = await handler.getParent(Number(parentId));
```

Comment:
i would move all this to be in a function called - validation.
The validation function will return {isValid: true / false, message?: ''}
and then the route will check if it got isValid false. If so then it will send 400 and the message.

Reply:
i moved it out but the number of rows remains the same so I think we haven't gained anything. do you agree?

Reply:
I don't see you created the validation function